megamek/src/megamek/server/totalWarfare/TWGameManager.java

```diff
@@ -19198,7 +19198,10 @@ private Vector<Report> applyMekSystemCritical(Entity en, int loc, CriticalSlot c
                         if (game.getOptions().booleanOption(OptionsConstants.PLAYTEST_2)) {
                             // No PSR for HD Gyro hits.
                         } else {
-                            game.addPSR(new PilotingRollData(en.getId(), 2, "gyro hit"));
+                            // Core ruleset: First hit to HD gyro does not require PSR per errata
+                            if (en.getGyroType() != Mek.GYRO_HEAVY_DUTY) {
```

Comment:
This seems ok to me. But the playtest code and comments above that are strange. They seem to make the assumption that this whole code block is only reached for HD (assuming this means heavy duty) gyros. But I don't think it is or we wouldn't need the present change. Also I couldn't see anything in the code above to make it limited to HD gyros.